src/promptflow-core/promptflow/executor/flow_executor.py

```diff
@@ -209,7 +209,11 @@ def create(
         :return: A new instance of FlowExecutor.
         :rtype: ~promptflow.executor.flow_executor.FlowExecutor
         """
-        setup_exporter_from_environ()
+        only_inject_openai = kwargs.get("only_inject_openai", False)
+        if only_inject_openai:
+            inject_openai_api()
+        else:
+            setup_exporter_from_environ()
```

Comment:
shall this branch do: 1. inject openai  2 set exporter?

Reply:
we didn't modify the existing func, it already calls inject